src/Workspaces/Core/Portable/Shared/Utilities/DocumentationComment.cs

```diff
@@ -328,6 +328,14 @@ private void ParseCallback(XmlReader reader)
             return text;
         }
 
+        public DocumentationComment? GetParameter(string parameterName)
```

Comment:
I should add some better doc comments here -- but to simplify handling in the other parts of the code, the idea here is if we have a <params> member, we can create a new DocumentationComment object that has a <summary> tag with the same contents, so everything else can treat that identically.